app/src/main/java/com/kickstarter/services/interceptors/GraphQLInterceptor.kt

```diff
@@ -1,17 +1,22 @@
 package com.kickstarter.services.interceptors
 
+import com.kickstarter.libs.Build
 import com.kickstarter.libs.CurrentUserType
 import okhttp3.Interceptor
 import okhttp3.Response
 
-class GraphQLInterceptor(private val clientId: String, private val currentUser: CurrentUserType) : Interceptor {
+class GraphQLInterceptor(private val clientId: String,
+                         private val currentUser: CurrentUserType,
+                         private val build: Build) : Interceptor {
     override fun intercept(chain: Interceptor.Chain?): Response {
         val original = chain!!.request()
         val builder = original.newBuilder().method(original.method(), original.body())
-        if (this.currentUser.exists()) {
-            builder.addHeader("Authorization", "token " + this.currentUser.accessToken)
+        val accessToken = this.currentUser.accessToken
+        if (accessToken != null) {
+            builder.addHeader("Authorization", "token $accessToken")
         }
-        builder.addHeader("X-KICKSTARTER-CLIENT", this.clientId)
+        builder.addHeader("User-Agent", WebRequestInterceptor.userAgent(this.build))
```

Comment:
Is the output value of `this.build` usually `Kickstarter Android Mobile Variant/internalDebug ...`?

If there is time, we could add a quick unit test to enforce that this is always passed in.

Reply:
ooh yeah I don't think there's any tests test for this! i can add one!

Reply:
so i wrote a test to test the value but to test the actual interceptor i'd need to add a new dependency https://github.com/square/okhttp/tree/master/mockwebserver which i think we should but not in this PR, i'm going to ask Jamie to file a separate JIRA ticket for it

Reply:
Ah gotcha. The test looks good, and mocking out the web server does seem like a separate task so I agree, LGTM 👍